docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  slackbot-ai:
+    container_name: slack-bot-ai
+    image: slackbot-ai:latest
+    build: .
+    ports:
+      - "7999:7999"
```

Comment:
7999 was completely arbitrary, just not to clash with usual django app at 8000. The app port can be parameterized by the env, do you think this is useful to have?

Reply:
Done!